php/7.3/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM php:7.3-fpm-alpine
+
+MAINTAINER JG <julien@mangue.eu>
+
+RUN apk add --no-cache --virtual .build-deps \
+        $PHPIZE_DEPS \
+        curl-dev \
+        imagemagick-dev \
+        libtool \
+        libxml2-dev \
+        postgresql-dev \
+        sqlite-dev \
+    && apk add --no-cache \
+        curl \
+        git \
+        imagemagick \
+        mysql-client \
+        postgresql-libs \
+        libintl \
+        icu \
+        icu-dev \
+        libzip-dev \
```

Comment:
The only noteworthy change is that PHP 7.3's `zip` needs this libzip-dev package in order to work.